pages/common/kubectl-delete.md

```diff
@@ -29,4 +29,4 @@
 
 - Delete resources defined in a YAML manifest:
 
-`kubectl delete --filename {{path/to/manifest.yaml}}`
+`kubectl delete --filename {{path/to/manifest}}.yaml`
```

Comment:
The extesion should be outside the token syntax only if it's mandatory. If you can save the YAML in a `.txt` file should be inside the token syntax.

Reply:
I am not sure about this command other pages had .yaml outside so I made this PR.

Reply:
Maybe it's a bug in the page. @bl-ue do you remember?

Reply:
Adding the pages having `.yaml` outside for reference.

[kubectl-describe](https://github.com/tldr-pages/tldr/blob/main/pages/common/kubectl-describe.md)
[kubectl-get](https://github.com/tldr-pages/tldr/blob/main/pages/common/kubectl-get.md)

Reply:
@navarroaxel I'm not sure. However, I tested this command invocation and it appears that `kubectl delete --filename` can take any valid filepath. So @kbdharun in this case the `.yaml` extension should actually be inside the `{{ }}` token syntax:
```suggestion
`kubectl delete --filename {{path/to/manifest.yaml}}`
```
And yes, @kbdharun `kubectl-describe` and `kubectl-get` should both be updated as well; I tested and found their behavior to be the same as here. I think just add them to this PR.